.github/workflows/ci.yml

```diff
@@ -154,13 +154,13 @@ jobs:
     runs-on: ubuntu-16.04
     strategy:
       matrix:
-        variant: ['gcc-5.5', 'clang-3.8']
+        variant: ['gcc-5', 'clang-3.8']
     steps:
       - name: Install prerequisites 📦
         run:  |
           sudo add-apt-repository ppa:ubuntu-toolchain-r/test
           sudo apt-get update
-          sudo apt-get install ninja-build
+          sudo apt-get install ninja-build gcc-5 g++-5
```

Comment:
You can remove the install of gcc/g++-5, since ubuntu-toolchains installs 5.5 for 16.04. This is the same as in the base image of github actions.

https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/test
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1604-README.md